src/metabase/integrations/slack.clj

```diff
@@ -130,11 +136,7 @@
 (defn users-list
   "Calls Slack API `users.list` endpoint and returns the list of available users."
   [& {:as query-parameters}]
-  (->> (paged-list-request "users.list" :members query-parameters)
-       ;; filter out deleted users and bots. At the time of this writing there's no way to do this in the Slack API
-       ;; itself so we need to do it after the fact.
-       (filter (complement :deleted))
-       (filter (complement :is_bot))))
+  [])
```

Comment:
this would mean disabling setting pulse to individual users

Reply:
I thought we agree on that. I can enable it back

Reply:
Ideally we want to disable user-list, except the current user

Reply:
If your patch works with keeping user-list, I think we should enable it back.

Reply:
I don't see an easy way to do so, since we don't have this info in the current request from UI to backend.

I think it's fine that we disable pulse to users, I will mention that in the announcement. Old user pulses would still work
We have 5 user pulses against 50 channel last month.

If we will get a lot of requests to enable them back, won't be hard to do so.

Reply:
I agree

Reply:
Thank you for raising your concern!